modules/library/cql/src/test/java/org/geotools/filter/text/cql2/FilterToCQLTest.java

```diff
@@ -283,6 +286,49 @@ public void testContains() throws Exception {
 	public void testTouches() throws Exception {
 		cqlTest("TOUCHES(theGeom, POLYGON ((0 0, 0 10, 10 10, 10 0, 0 0)))");
 	}
+	
+    
+    @Ignore // Parser doesn't implement this
+    @Test
+    public void testAttributeContainsQuote() throws Exception {
+        String cql = "INTERSECTS(\"the\"\"geom\", POINT (1 2))";
+        Filter filter = CQL.toFilter(cql);
+        Assert.assertNotNull( filter );
+        
+        // double quote escaped by repeating should be unescaped to just one
+        Assert.assertEquals("the\"geom",((PropertyName)(((Intersects) filter).getExpression1())).getPropertyName());
+        
+        FilterToCQL toCQL = new FilterToCQL();
+        String output = filter.accept( toCQL, null ).toString();
+        // Should be escaped again
+        Assert.assertEquals( cql,output );
+    }
+    
+    @Ignore // Parser doesn't implement this
+    @Test
+    public void testAttributeContainsSpace() throws Exception {
+        cqlTest("INTERSECTS(\"the geom\", POINT (1 2))");
+    }
+    
+    @Test
+    public void testAttributeContainsOperator() throws Exception {
+        cqlTest("INTERSECTS(\"the-geom\", POINT (1 2))");
+    }
+    
+    @Test
+    public void testAttributeContainsComma() throws Exception {
+        cqlTest("INTERSECTS(\"the,geom\", POINT (1 2))");
```

Comment:
For whatever reason I usually delimiters with single quotes (probably just so the java string representation reads better.

```
cqlTest("INTERSECTS('the,geom', POINT (1 2))");
```

I am sure this is just what I am used to.